### ISSUE TITLE
update-center2 job being unstable

### ISSUE THREAD

**Issue by Wadeck** [jenkins-infra/helpdesk#2747]
### Service

ci.jenkins.io

### Summary

**Where**
PR on update-center2

**What**
In a matter of 30 minutes I got 1 success, 1 failure to to a timeout, 1 failure to due lack of JAVA_HOME.

### Reproduction steps

1) Timeout
https://ci.jenkins.io/blue/organizations/jenkins/Infra%2Fupdate-center2/detail/PR-558/4/pipeline/
https://github.com/jenkins-infra/update-center2/runs/4850789901

2) Success (on master after merge)
https://github.com/jenkins-infra/update-center2/runs/4850804919

3) Missing env
https://github.com/jenkins-infra/update-center2/runs/4851078590

<details>

<summary>Screenshots</summary>

![Screenshot_2022-01-18_102423_001](https://user-images.githubusercontent.com/2662497/149923937-e7580fd7-6f53-48a1-967b-2fa9bd476e68.png)

![Screenshot_2022-01-18_095910_001](https://user-images.githubusercontent.com/2662497/149923945-2c860857-b95a-4728-ba9b-4b97d1ee4f70.png)

</details>

**Comment by timja** [comment]
related to https://github.com/jenkins-infra/helpdesk/issues/2732

**Comment by MarkEWaite** [comment]
As far as I can tell, the lack of `JAVA_HOME` was due to a configuration issue for the global tool installer named `jdk8`.  The agent that is being used (label `linux`) has Java 8 already installed on the agent image at `/opt/jdk-8`.  I modified the tool installer like this:

![Screenshot 2022-01-24 150924](https://user-images.githubusercontent.com/156685/150873313-d31cb6cc-5ee5-4286-bcb5-0646d501f889.png)

However, that change was made interactively.  I'll work with @lemeurherve tomorrow to make that configuration persist permanently in the ci.jenkins.io configuration.
